vaadin-grid-pro-flow-parent/vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/gridProConnector.js

```diff
@@ -34,11 +45,20 @@ window.Vaadin.Flow.gridProConnector = {
       root.appendChild(component);
       this._grid._cancelStopEdit();
       component.focus();
+
+      component.__itemKey = rowData.item.key;
+      this._grid.toggleAttribute(LOADING_EDITOR_CELL_ATTRIBUTE, true);
     };
 
     // Not needed in case of custom editor as value is set on server-side.
     // Overridden in order to avoid blinking of the cell content.
     column._setEditorValue = function (editor, value) {};
+
+    const stopCellEdit = column._stopCellEdit;
+    column._stopCellEdit = function (editor, itemKey, grid) {
+      stopCellEdit.call(this, editor, itemKey, grid);
+      this._grid.toggleAttribute(LOADING_EDITOR_CELL_ATTRIBUTE, false);
+    };
```

Comment:
The arguments here are different than what is actually in the web component: https://github.com/vaadin/web-components/blob/bafcfd7d3b9c02af6162444666bbad1f5488b225/packages/grid-pro/src/vaadin-grid-pro-edit-column-mixin.js#L295-L302

Could use `apply` instead to keep this logic safe for future changes in the WC.

Reply:
Done